dataprofiler/tests/profilers/test_unstructured_text_profile.py

```diff
@@ -166,6 +166,155 @@ def test_case_sensitivity(self):
                                    'friends': 1}
             self.assertDictEqual(expected_word_count, profile['word_count'])
 
+    def test_merge_most_common_words_count(self):
+        ### default values of most common words for both profiles
+        text_profile1 = TextProfiler("Name")
+        text_profile1._stop_words = []  # set stop_words to empty list for easy inspection
+        sample1 = pd.Series(["this is test,", " this is a test sentence"])
+        text_profile1.update(sample1)
+
+        text_profile2 = TextProfiler("Name")
+        text_profile2._stop_words = []  # set stop_words to empty list for easy inspection
+        sample2 = pd.Series(["this is", "this"])
+        text_profile2.update(sample2)
+
+        text_profile3 = text_profile1 + text_profile2
+        profile = text_profile3.profile
+
+        # as merged profile's word_count length is None, it is set to
+        # the length of the merged word_count, which is 5
+        expected_word_count = {'this': 4, 'is': 3, 'test': 2, 'a': 1, 'sentence': 1}
+        self.assertDictEqual(expected_word_count, profile["word_count"])
+
+        ### one profile has default values of most common words
+        ### the other profile has it set
+        text_profile1 = TextProfiler("Name")
+        text_profile1._num_most_common_words = 3
+        text_profile1._stop_words = []  # set stop_words to empty list for easy inspection
+        sample1 = pd.Series(["this is test,", " this is a test sentence"])
+        text_profile1.update(sample1)
```

Comment:
do we need to re-profile or can we use the above and alter the most_common?

Reply:
same for below

Reply:
we don't actually need to profile again. I updated this.